services/workflows-service/prisma/migrations/20250129142137_documents_init/migration.sql

```diff
@@ -0,0 +1,81 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `documents` on the `Business` table. All the data in the column will be lost.
+
+*/
+-- CreateEnum
+CREATE TYPE "DocumentStatus" AS ENUM ('provided', 'unprovided', 'requested');
+
+-- CreateEnum
+CREATE TYPE "DocumentDecision" AS ENUM ('approved', 'rejected', 'revisions');
```

Comment:
Just making sure we are aligned, in the revision flow what will happen is:

1. user upload doc 1
2. agent will ask for reupload
3. status changed to revisions on doc 1
4. user upload another doc, we create doc 2 (same category and so on, but we bump the version)
5. now we have doc 1 (v1) still in revision (which is okay i think because its kind of an audit log as well), and doc 2 (v2) in blank/new state